nixos/tests/matrix/draupnir.nix

```diff
@@ -0,0 +1,239 @@
+{
+  pkgs,
```

Comment:
```suggestion
  pkgs,
  withPantalaimon ? false
```

Can we make the pantalaimon parts optional? 

Since this test was written libolm has been marked vulnerable due to various issues around its maintenance, and we want the test to be built by hydra or r-ryantm, which will not happen if it transitively depends on libolm.

This would mean to make all the pantalaimon specific configuration conditional on that variable.